### DISCUSSION TITLE
Ist die Ref-Umgebung offline?

### DISCUSSION THREAD

**Discussion by msteffenDev** [Digitaler-Impfnachweis/certification-apis#202] [Q&A]
Ich erhalte folgenden Fehler:

 0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0*   Trying 10.30.17.10...
* TCP_NODELAY set
* Connected to id.ru.impfnachweis.info (10.30.17.10) port 443 (#0)
* ALPN, offering h2
* ALPN, offering http/1.1
* Cipher selection: ALL:!EXPORT:!EXPORT40:!EXPORT56:!aNULL:!LOW:!RC4:@STRENGTH
* successfully set certificate verify locations:
*   CAfile: /etc/ssl/cert.pem
  CApath: none
* TLSv1.2 (OUT), TLS handshake, Client hello (1):
} [512 bytes data]
* TLSv1.2 (IN), TLS handshake, Server hello (2):
{ [64 bytes data]
* TLSv1.2 (IN), TLS handshake, Certificate (11):
{ [5065 bytes data]
* TLSv1.2 (IN), TLS handshake, Server key exchange (12):
{ [657 bytes data]
* TLSv1.2 (IN), TLS handshake, Server finished (14):
{ [4 bytes data]
* TLSv1.2 (OUT), TLS handshake, Client key exchange (16):
} [138 bytes data]
* TLSv1.2 (OUT), TLS change cipher, Client hello (1):
} [1 bytes data]
* TLSv1.2 (OUT), TLS handshake, Finished (20):
} [16 bytes data]
* TLSv1.2 (IN), TLS change cipher, Client hello (1):
{ [1 bytes data]
* TLSv1.2 (IN), TLS handshake, Finished (20):
{ [16 bytes data]
* SSL connection using TLSv1.2 / ECDHE-RSA-AES128-GCM-SHA256
* ALPN, server accepted to use h2
* Server certificate:
*  subject: 1.3.6.1.4.1.311.60.2.1.3=DE; 1.3.6.1.4.1.311.60.2.1.2=Nordrhein-Westfalen; 1.3.6.1.4.1.311.60.2.1.1=Koeln; businessCategory=Private Organization; serialNumber=HRB 82994; C=DE; ST=Nordrhein-Westfalen; L=Koeln; postalCode=50670; street=Im Mediapark 5; O=ubirch GmbH; CN=certify.demo.ubirch.com
*  start date: Aug 10 10:45:46 2021 GMT
*  expire date: Aug 13 10:45:46 2022 GMT
*  subjectAltName: host \"id.ru.impfnachweis.info\" matched cert's \"id.ru.impfnachweis.info\"
*  issuer: C=DE; O=D-Trust GmbH; CN=D-TRUST SSL Class 3 CA 1 EV 2009
*  SSL certificate verify ok.
* Using HTTP2, server supports multi-use
* Connection state changed (HTTP/2 confirmed)
* Copying HTTP/2 data in stream buffer to connection buffer after upgrade: len=0
* Using Stream ID: 1 (easy handle 0x7f8d7e80e800)
> GET /auth/realms/ubirch-certify/protocol/openid-connect/auth?redirect_uri=connector://authenticated&response_type=code&scope=openid&client_id=user-access-ti&nonce=785302914 HTTP/2
> Host: id.ru.impfnachweis.info
> User-Agent: curl/7.54.0
> Accept: */*
> 
* Connection state changed (MAX_CONCURRENT_STREAMS updated)!

  0     0    0     0    0     0      0      0 --:--:--  0:00:01 --:--:--     0
  0     0    0     0    0     0      0      0 --:--:--  0:00:02 --:--:--     0
  0     0    0     0    0     0      0      0 --:--:--  0:00:03 --:--:--     0
  0     0    0     0    0     0      0      0 --:--:--  0:00:04 --:--:--     0
  0     0    0     0    0     0      0      0 --:--:--  0:00:05 --:--:--     0
  0     0    0     0    0     0      0      0 --:--:--  0:00:06 --:--:--     0
  0     0    0     0    0     0      0      0 --:--:--  0:00:07 --:--:--     0
  0     0    0     0    0     0      0      0 --:--:--  0:00:08 --:--:--     0
  0     0    0     0    0     0      0      0 --:--:--  0:00:09 --:--:--     0
  0     0    0     0    0     0      0      0 --:--:--  0:00:10 --:--:--     0
  0     0    0     0    0     0      0      0 --:--:--  0:00:11 --:--:--     0
  0     0    0     0    0     0      0      0 --:--:--  0:00:12 --:--:--     0
  0     0    0     0    0     0      0      0 --:--:--  0:00:13 --:--:--     0
  0     0    0     0    0     0      0      0 --:--:--  0:00:14 --:--:--     0
  0     0    0     0    0     0      0      0 --:--:--  0:00:15 --:--:--     0
  0     0    0     0    0     0      0      0 --:--:--  0:00:16 --:--:--     0
  0     0    0     0    0     0      0      0 --:--:--  0:00:17 --:--:--     0
  0     0    0     0    0     0      0      0 --:--:--  0:00:18 --:--:--     0
  0     0    0     0    0     0      0      0 --:--:--  0:00:19 --:--:--     0
  0     0    0     0    0     0      0      0 --:--:--  0:00:20 --:--:--     0
  0     0    0     0    0     0      0      0 --:--:--  0:00:21 --:--:--     0
  0     0    0     0    0     0      0      0 --:--:--  0:00:22 --:--:--     0
  0     0    0     0    0     0      0      0 --:--:--  0:00:23 --:--:--     0
  0     0    0     0    0     0      0      0 --:--:--  0:00:24 --:--:--     0
  0     0    0     0    0     0      0      0 --:--:--  0:00:25 --:--:--     0
  0     0    0     0    0     0      0      0 --:--:--  0:00:26 --:--:--     0
  0     0    0     0    0     0      0      0 --:--:--  0:00:27 --:--:--     0
  0     0    0     0    0     0      0      0 --:--:--  0:00:28 --:--:--     0
  0     0    0     0    0     0      0      0 --:--:--  0:00:29 --:--:--     0< HTTP/2 504 
< strict-transport-security: max-age=15552000; includeSubDomains; preload
< content-type: text/plain; charset=utf-8
< content-length: 15
< date: Wed, 01 Dec 2021 13:37:58 GMT
< 
{ [15 bytes data]
Gateway Timeout
100    15    0    15    0     0      0      0 --:--:--  0:00:30 --:--:--     3
100    15    0    15    0     0      0      0 --:--:--  0:00:30 --:--:--     3
* Connection #0 to host id.ru.impfnachweis.info left intact

**Comment by jweko** [comment]
Wir erhalten ebenfalls ein Gateway timeout bei Versuch in der RU etwas aufzurufen/Zertifikate zu erstellen.

**Reply by thinkberg** [reply]
Wir versuchen das Problem einzugrenzen. Besteht die Moeglichkeit ein Log zu bekommen von so einem Zugriff, am besten auch mit Zeitstempeln? Bitte an dcc-support@ubirch.com senden. Das muss hier nicht rumliegen.

**Reply by marcusthierfelder** [reply]
Gibt es schon Neuigkeiten? Wir bekommen immer noch ein Timeout.

**Comment by ManuelB** [comment]
Das Problem besteht bei uns auch. Hier ein paar Screenshots von unserem Status Tool und unser Log:
![Bildschirmfoto von 2021-12-14 09-03-44](https://user-images.githubusercontent.com/457641/145957475-f0b36067-8f0a-4e2c-b729-c5d5320cc7e1.png)
![Bildschirmfoto von 2021-12-14 09-03-39](https://user-images.githubusercontent.com/457641/145957485-eee75f66-23a3-4405-99ac-5e9777d15fa7.png)

```
$ mvn quarkus:dev
WARNING: An illegal reflective access operation has occurred
WARNING: Illegal reflective access by com.google.inject.internal.cglib.core.$ReflectUtils$1 (file:/usr/share/maven/lib/guice.jar) to method java.lang.ClassLoader.defineClass(java.lang.String,byte[],int,int,java.security.ProtectionDomain)
WARNING: Please consider reporting this to the maintainers of com.google.inject.internal.cglib.core.$ReflectUtils$1
WARNING: Use --illegal-access=warn to enable warnings of further illegal reflective access operations
WARNING: All illegal access operations will be denied in a future release
[INFO] Scanning for projects...
[INFO] 
[INFO] -----------------------< health.ere:ere-ps-app >------------------------
[INFO] Building ere-ps-app 1.0.0-SNAPSHOT
[INFO] --------------------------------[ jar ]---------------------------------
[WARNING] The POM for com.sun.xml.ws:jaxws-rt:jar:2.2.10 is invalid, transitive dependencies (if any) will not be available, enable debug logging for more details
[INFO] 
[INFO] --- quarkus-maven-plugin:1.13.1.Final:dev (default-cli) @ ere-ps-app ---
Downloading from project-repo: file:///home/manuel/git/dgc-ps-app/project-repo/health/ere/ere-ps-app/1.0.0-SNAPSHOT/maven-metadata.xml
[INFO] Using 'UTF-8' encoding to copy filtered resources.
[INFO] Copying 20 resources
[INFO] Nothing to compile - all classes are up to date
Listening for transport dt_socket at address: 5005
__  ____  __  _____   ___  __ ____  ______ 
 --/ __ \/ / / / _ | / _ \/ //_/ / / / __/ 
 -/ /_/ / /_/ / __ |/ , _/ ,< / /_/ /\ \   
--\___\_\____/_/ |_/_/|_/_/|_|\____/___/   
2021-12-14 09:01:49,310 INFO  [hea.ere.ps.ser.dgc.DigitalGreenCertificateService] (Quarkus Main Thread) Application started go to: http://localhost:8080/dgc/covid-19-vaccination-certificate.html
2021-12-14 09:01:49,315 INFO  [hea.ere.ps.ser.dgc.DigitalGreenCertificateService] (Quarkus Main Thread) idp.client.id: user-access-ti
2021-12-14 09:01:49,315 INFO  [hea.ere.ps.ser.dgc.DigitalGreenCertificateService] (Quarkus Main Thread) idp.base.url: https://id.ru.impfnachweis.info/auth/realms/bmg-ti-certify/
2021-12-14 09:01:49,315 INFO  [hea.ere.ps.ser.dgc.DigitalGreenCertificateService] (Quarkus Main Thread) digital-green-certificate-service.issuerAPIUrl: https://api.ru.impfnachweis.info
2021-12-14 09:01:49,389 INFO  [io.quarkus] (Quarkus Main Thread) ere-ps-app 1.0.0-SNAPSHOT on JVM (powered by Quarkus 1.13.1.Final) started in 1.627s. Listening on: http://0.0.0.0:8080
2021-12-14 09:01:49,390 INFO  [io.quarkus] (Quarkus Main Thread) Profile dev activated. Live Coding activated.
2021-12-14 09:01:49,390 INFO  [io.quarkus] (Quarkus Main Thread) Installed features: [cdi, rest-client, resteasy, resteasy-jsonb, servlet]
2021-12-14 09:02:02,755 WARNING [hea.ere.ps.ser.com.sec.SecretsManagerService] (executor-thread-2) Created SSLContext will accept all server certificates without validation
2021-12-14 09:02:03,475 WARNING [hea.ere.ps.ser.com.sec.SecretsManagerService] (executor-thread-2) Created SSLContext will accept all server certificates without validation
```

**Reply by marcusthierfelder** [reply]
Zwischenzeitlich ging es wieder. Im laufe des gestrigen Nachmittags kam es dann zu Problemen beim TokenExchange

**Comment by ManuelB** [comment]
@thinkberg Ich habe das auch an dcc-support@ubirch.com gesendet. Die Herausforderungen auf GitHub gemeinsam lösen würde deutlich besser skalieren, als dass 30 - 100 verschiedene Organisationen einzeln E-Mails an dcc-support@ubirch.com senden.

Wir versorgen 5 verschiedene Systeme mit unserer Software, für mich wäre es hier sehr hilfreich, wenn ich einfach auf die GitHub Diskussion verweisen könnte.

**Reply by thinkberg** [reply]
Sieht so aus als sei ein GEMATIK Service nicht verfuegbar, der fuer die Funktion notwendig ist. Schauen wir uns an.

**Reply by ManuelB** [reply]
@thinkberg möglicherweise wurde dieser auf Grund von Log4Shell runter gefahren:
https://fachportal.gematik.de/ti-status#c4722

Wenn Du die Handynummer von Gematik Leuten brauchst, sag Bescheid.

**Reply by ManuelB** [reply]
@thinkberg auch ein großes Dankeschön an Dich, dass Du hier so offen mit uns kommunizierst. Ich hoffe, dass sich daran einige Leute in höheren Positionen ein Beispiel nehmen, dass man besonders dann Dinge schnell zum Laufen bekommt, wenn man offen auf arbeitsebene miteinander redet.

**Reply by marcusthierfelder** [reply]
vielen dank.

**Reply by thinkberg** [reply]
Der Ausfall wurde uns bestaetigt. Die Gematik arbeitet an einer Behebung. Konkret handelt es sich um einen OCSP Responder, wir arbeiten jetzt daran, dass wir das gemeinsam schneller detektieren koennen. Der Ausfall begann ca. 8:50. Ich melde mich, sobald das Problem behoben ist.

**Reply by marcusthierfelder** [reply]
Ich vermute mal das es da noch keine Neuigkeiten gibt? Zumindest haben wir noch immer die gleichen Fehler.

**Comment by ManuelB** [comment]
@marcusthierfelder wir bekommen aktuell auch einen Fehler vom IdP:

```
2021-12-14 09:19:33,339 INFO  [hea.ere.ps.ser.idp.cli.IdpClient] (executor-thread-1) Initializing using url: https://id.ru.impfnachweis.info/auth/realms/bmg-ti-certify//.well-known/openid-configuration
2021-12-14 09:19:34,420 WARNING [hea.ere.ps.ser.com.sec.SecretsManagerService] (executor-thread-1) Created SSLContext will accept all server certificates without validation
2021-12-14 09:19:35,054 INFO  [hea.ere.ps.ser.con.cer.CardCertificateReaderService] (executor-thread-1) Certificate list size = 1
2021-12-14 09:19:36,848 WARNING [hea.ere.ps.ser.idp.IdPService] (executor-thread-1) Idp login did not work: health.ere.ps.exception.idp.IdpClientException: Unexpected response from id.ru.impfnachweis.info
	at health.ere.ps.service.idp.client.IdpClient.login(IdpClient.java:324)
	at health.ere.ps.service.idp.client.IdpClient.login(IdpClient.java:218)
	at health.ere.ps.service.idp.client.IdpClient_Subclass.login$$superaccessor16(IdpClient_Subclass.zig:2965)
	at health.ere.ps.service.idp.client.IdpClient_Subclass$$function$$16.apply(IdpClient_Subclass$$function$$16.zig:59)
	at io.quarkus.arc.impl.AroundInvokeInvocationContext.proceed(AroundInvokeInvocationContext.java:54)
	at io.quarkus.arc.runtime.devconsole.InvocationInterceptor.proceed(InvocationInterceptor.java:63)
	at io.quarkus.arc.runtime.devconsole.InvocationInterceptor.monitor(InvocationInterceptor.java:49)
	at io.quarkus.arc.runtime.devconsole.InvocationInterceptor_Bean.intercept(InvocationInterceptor_Bean.zig:521)
	at io.quarkus.arc.impl.InterceptorInvocation.invoke(InterceptorInvocation.java:41)
	at io.quarkus.arc.impl.AroundInvokeInvocationContext.perform(AroundInvokeInvocationContext.java:41)
	at io.quarkus.arc.impl.InvocationContexts.performAroundInvoke(InvocationContexts.java:32)
	at health.ere.ps.service.idp.client.IdpClient_Subclass.login(IdpClient_Subclass.zig:2887)
	at health.ere.ps.service.idp.IdPService.requestBearerToken(IdPService.java:105)
	at health.ere.ps.service.idp.IdPService_Subclass.requestBearerToken$$superaccessor2(IdPService_Subclass.zig:340)
	at health.ere.ps.service.idp.IdPService_Subclass$$function$$2.apply(IdPService_Subclass$$function$$2.zig:33)
	at io.quarkus.arc.impl.AroundInvokeInvocationContext.proceed(AroundInvokeInvocationContext.java:54)
	at io.quarkus.arc.runtime.devconsole.InvocationInterceptor.proceed(InvocationInterceptor.java:63)
	at io.quarkus.arc.runtime.devconsole.InvocationInterceptor.monitor(InvocationInterceptor.java:49)
	at io.quarkus.arc.runtime.devconsole.InvocationInterceptor_Bean.intercept(InvocationInterceptor_Bean.zig:521)
	at io.quarkus.arc.impl.InterceptorInvocation.invoke(InterceptorInvocation.java:41)
	at io.quarkus.arc.impl.AroundInvokeInvocationContext.perform(AroundInvokeInvocationContext.java:41)
	at io.quarkus.arc.impl.InvocationContexts.performAroundInvoke(InvocationContexts.java:32)
	at health.ere.ps.service.idp.IdPService_Subclass.requestBearerToken(IdPService_Subclass.zig:297)
	at health.ere.ps.service.idp.IdPService_Observer_requestBearerToken_0261c8c58fae7be81975d0a143026a6569dfd3cc.notify(IdPService_Observer_requestBearerToken_0261c8c58fae7be81975d0a143026a6569dfd3cc.zig:147)
	at io.quarkus.arc.impl.EventImpl$Notifier.notifyObservers(EventImpl.java:283)
	at io.quarkus.arc.impl.EventImpl$Notifier.notify(EventImpl.java:264)
	at io.quarkus.arc.impl.EventImpl.fire(EventImpl.java:70)
```

**Comment by ManuelB** [comment]
Falls noch jemand eine Meldung an dcc-support@ubirch.com ihr könnt euch gerne auf meinen Vorgang DCC-1806 beziehen.

**Comment by thinkberg** [comment]
Solange die Gematik den Service nicht wieder aktiviert sind wir da etwas gehindert.
Ich frage aber nochmal nach.

-- 
Matthias L. Jugel | @thinkberg

**Comment by thinkberg** [comment]
Also, Rueckmeldung ist jetzt, dass der OCSP responder wieder online ist. Es sollte also egentlich funktionieren.

**Comment by ManuelB** [comment]
Geht bei uns leider immer noch nicht:
![Bildschirmfoto von 2021-12-22 20-34-38](https://user-images.githubusercontent.com/457641/147145980-124fb97a-b026-441f-98dd-bd0230202200.png)
![Bildschirmfoto von 2021-12-22 20-34-28](https://user-images.githubusercontent.com/457641/147145985-304d4576-3104-4554-89cc-4f9cf8b5716d.png)

**Comment by JDoodsCGM** [comment]
Dear sender

due to a ransomware attack and necessary security measures, CGM is temporarily not reachable via the usual eMail addresses and Office phone numbers.
Please feel free to use SMS and mobile telephony to contact us.

Thank you for your understanding and best regards, your CGM Team

-----------------

Sehr geehrte Absenderin,
sehr geehrter Absender,

aufgrund einer Ransomware-Attacke und notwendiger Sicherheitsmaßnahmen, ist CGM vorübergehend nicht über die gewohnten eMail-Adressen und Festnetznummern erreichbar.
Bitte nutzen Sie gerne SMS und Mobilfunk-Telefonie, um mit uns Kontakt aufzunehmen.

Vielen Dank für Ihr Verständnis und freundliche Grüße, Ihr CGM-Team

**Comment by marcusthierfelder** [comment]
Hallo

leider ist  api.ru.impfnachweis.info  immer noch nicht erreichbar (fast ein Monat). Ist bekannt wer den Server betreibt oder was genau das Problem ist. Wir würden schon gern unsere Änderungen testen bevor die zum Kunden geht.

**Reply by ManuelB** [reply]
Hi Marcus, aktuelle Aussage ist dass ein OCSP Responder der Gematik nicht geht

**Reply by thinkberg** [reply]
Der OCSP Responder funktioniert wieder, daran kann es nicht liegen. Die Fehlermeldung ist mir etwas unklar. Ist es "nicht erreichbar" im Sinne der Netzwerkverbindung, oder ein Fehler, wie 40x? Es kann durchaus auch an Routen Einstellungen liegen, auf die wir keinen Einfluss haben. Wir hatten öfter mal falsch konfigurierte Konrektoren.

**Reply by marcusthierfelder** [reply]
Wir bekommen je nach Implementierung andere Fehler (sicher alles das gleiche):

a) anpingen geht nicht (hier weiss ich leider nicht ob schon deaktiviert war)
ping api.ru.impfnachweis.info
PING api.ru.impfnachweis.info (10.30.17.10): 56 data bytes
Request timeout for icmp_seq 0
...

b) "python" sagt: 
DEBUG:urllib3.connectionpool:https://api.ru.impfnachweis.info:443 "POST /api/certify/v2/issue HTTP/1.1" 504 15
nope: 504  Gateway Timeout

c) "java" sagt:
09:41:19.829 [main] DEBUG org.apache.http.impl.conn.PoolingHttpClientConnectionManager - Connection released: [id: 3][route: {s}->https://id.impfnachweis.info:443][total kept alive: 0; route allocated: 0 of 2; total allocated: 0 of 20]
2021-12-29 09:41:19,830: (com.zol.med.imp.ImpfZertPDFGenerator:109) INFO | 09:41:19.829 [main] ERROR de.impfdoc.impfzert.eu.EuCertService - java.net.NoRouteToHostException: No route to host

Jetzt könnte das natürlich am Konnektor/Netzwerk/Route zum Konnektor ... da haben wir aber nichts umgestellt / andere haben den gleichen Fehler.

**Reply by thinkberg** [reply]
Das hilft schon mal, danke.

**Reply by thinkberg** [reply]
So, das Problem sollte jetzt behoben sein. Es gab wohl noch Netzwerkregeln, die erst nach einer Änderung griffen und das ganze hat sich auch noch mit dem OCSP Responder überschnitten.

**Reply by marcusthierfelder** [reply]
Vielen Dank, es geht jetzt.

**Reply by ManuelB** [reply]
Habs auch gerade nochmals probiert. Geht immer noch ;-)

**Comment by jweko** [comment]


**Reply by Ein-Tim** [reply]
@jweko Could you please make sure that this comment is only posted if necessary. Thank you.

**Comment by ruiyan-yu** [comment]
Hi,

API Endpoint für die Erstellung von PDF ist offline. Error 500
response(61): { [5 bytes data]
response(62): * TLSv1.3 (IN), TLS handshake, Newsession Ticket (4):
response(63): { [130 bytes data]
response(64): * We are completely uploaded and fine
response(65): { [5 bytes data]
response(66): * Connection state changed (MAX_CONCURRENT_STREAMS == 250)!
response(67): } [5 bytes data]
response(68): < HTTP/2 500

Mit freundlichen Grüßen

Duria e.G

**Reply by ManuelB** [reply]
Wir bekommen zwar einen 501. Ich kann aber bestätigen, dass es nicht mehr funktioniert:

Request von unserem Frontend zu unserem Server (wird im Server mit einem Token erweitert)

POST /api/certify/v2/issue HTTP/1.1
Host: localhost:8080
Connection: keep-alive
Content-Length: 189
sec-ch-ua: " Not A;Brand";v="99", "Chromium";v="96", "Google Chrome";v="96"
sec-ch-ua-mobile: ?0
User-Agent: Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/96.0.4664.110 Safari/537.36
sec-ch-ua-platform: "Linux"
Content-Type: application/json
Accept: */*
Origin: http://localhost:8080
Sec-Fetch-Site: same-origin
Sec-Fetch-Mode: cors
Sec-Fetch-Dest: empty
Referer: http://localhost:8080/dgc/covid-19-vaccination-certificate.html
Accept-Encoding: gzip, deflate, br
Accept-Language: de,en;q=0.9


HTTP/1.1 501 Not Implemented
Content-Type: application/json
Content-Length: 101

**Comment by ruiyan-yu** [comment]
Getest um 17:02. API Endpoint ist wieder online. Die Erstellung war erfolgreich. Danke

**Comment by thinkberg** [comment]
Bitte solche Probleme an dcc-support@ubirch.com melden. Ich bekomme nicht immer alles sofort mit, wenn hier was kommentiert wird.

**Comment by BettinaTH** [comment]
FYI: Endpoint ist wieder nicht erreichbar. Habe auch schon eine Mail an die oben genannte Adresse geschickt.

**Comment by kamianni** [comment]
Kann es sein, dass die Ref schon wieder down ist? (Mail ist verschickt :) )

Nachtrag 28.02.: Erzeugung der Zertifikate über Ref-Umgebung wieder möglich. Danke für den schnellen Fix!

**Reply by msteffenDev** [reply]
Bei uns funktioniert es immer noch nicht, Mail ist raus

**Reply by thinkberg** [reply]
Nur zur Sicherheit, weil das gestern ein Issue war. Ist der richtige Authentication Endpunkt konfiguriert: https://github.com/Digitaler-Impfnachweis/certification-apis/blob/master/SMCB-Authentication.md#official-endpoints

Es gab in der Liste einen den ich gestern entfernt habe, der schon seit langer Zeit als "deprecated" markiert war.

Leo.

**Reply by msteffenDev** [reply]
Hallo Herr Jugel,

das war die Lösung,

ich hatte noch den Endpoint https://id.ru.impfnachweis.info/auth/realms/ubirch-certify/protocol/openid-connect/auth definiert,

geändert in https://id.ru.impfnachweis.info/auth/realms/bmg-ti-certify/protocol/openid-connect/auth,

Abruf nun ok.

Vielen Dank!
